config/findbugs-exclude.xml

```diff
@@ -15,4 +15,11 @@
             <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
         </Or>
     </Match>
+    <Match>
+        <Or>
+            <Class name="~.*\.GsonAdapters.*"/>
+            <Class name="~.*\.Immutable.*"/>
+            <Class name="~.*\.Modifiable.*"/>
```

Comment:
Shouldn't there be a `Bug pattern` in here? Aren't we just suppressing everything for the classes as they are written now?